fixmydjango/fixmydjango
# Refs #35 - Track whether the error came from lib or from site

Link to task: https://github.com/fixmydjango/fixmydjango/issues/35
How to test:
- Run python manage.py migrate

- Submit a new error from your local site
- In the admin, you should see in the Error_Posts list a field called "Data came from" the value "Site"

-Submit a new error from the fixmydjango-lib
- In the admin, you should see in the Error_Posts list a field called "Data came from" the value "Lib"
